xapian-applications/omega/utils.cc

```diff
@@ -53,6 +53,9 @@ using namespace std;
 int
 string_to_int(const string &s)
 {
+    if (s[0] < '0' || s[0] > '9') {
+	cerr << "Value entered is not an integer\n";
+    }
```

Comment:
This will report errors in places it shouldn't - first one I found is that `$add` uses `string_to_int()` on its arguments, so `$add{-1,-2}` will report this twice, but I think it's likely there are many more.

Reply:
thanks i will fix that.